webpack.config.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+  // change to .tsx if necessary
+  entry: './src/app.jsx',
+  output: {
+    filename: './bundle.js',
+  },
+  resolve: {
+    // changed from extensions: [".js", ".jsx"]
+    extensions: ['.ts', '.tsx', '.js', '.jsx'],
+  },
+  module: {
+    rules: [
```

Comment:
we may need to add css configuration in webpack.config.
https://webpack.js.org/loaders/css-loader/